apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/SnifferConfigInitializer.java

```diff
@@ -18,6 +18,7 @@
 
 package org.apache.skywalking.apm.agent.core.conf;
 
+import lombok.Data;
```

Comment:
Don't use this.

Reply:
It is working before, please just refactor codes, rather than using new lib.

Reply:
I'm confused now. @kezhenxu94 mentioned that we should use `lombok` to avoid boilerplate codes(getters setters).

> @kezhenxu94 : [Use Lombok annotations to get rid of boilerplate codes (getters setters)](https://github.com/apache/skywalking/pull/4264#discussion_r367986163).

Reply:
@kezhenxu94 I think this should not use `Lombok`. Lombok is a good option for backend, but we don't need that at the agent side. 

Even we need to discuss whether we could accept this new lib, we should not ask in this pull request. @kezhenxu94 if you want this, please submit another pull request about this.